openapi/resource-server.yaml

```diff
@@ -650,6 +671,7 @@ paths:
       description: A client can fetch the latest state of an incoming payment to determine the amount received into the wallet address.
     parameters:
       - $ref: '#/components/parameters/id'
+      - $ref: '#/components/parameters/wallet-address'
```

Comment:
@sabineschaller Is it the case also for completing an incoming payment (https://github.com/interledger/open-payments/pull/305/files#diff-9debc2038853698222051b2c2a428cee93e9dda6af6076de559971933879580dR722-R725) since the path still contains that unique id?

Reply:
Yes, indeed.